### ISSUE TITLE
t/010_version.t fails on CentOS7

### ISSUE THREAD

**Issue by eserte** [dex4er/perluse#2]
I see the following failure:
```
$ perl5.24.1 -Mblib t/010_version.t    
1..1
script/perluse.sh: line 51: perlbrew: command not found
script/perluse.sh: line 55: /home/cpansand/perl5/perlbrew/etc/bashrc: No such file or directory
not ok 1
```

On other of my smoker systems (e.g. debian) /home/cpansand/perl5/perlbrew/etc/bashrc also does not exist, but there's no failure.

Maybe the difference is:
On a debian system:
```
$ sh -c 'echo $BASH_VERSION'

```
On a CentOS7 system:
```
$ sh -c 'echo $BASH_VERSION'
4.2.46(1)-release
```

**Comment by eserte** [comment]
The same failure happens also on my fedora28 smokers.

**Comment by dex4er** [comment]
Fixed in d60dc8db487338cb335a9992d578e0191a0eb743

**Comment by dex4er** [comment]
Released with https://github.com/dex4er/perluse/releases/tag/0.0201
